### PR TITLE
Add saving of blockchain. pr: #11

### PR DESCRIPTION
**Pull request by TheBestTvarynka**

### CODE REVIEW

**Comment by dmvict** [comment]
Good solution. 

What do you think about simpler version?

File: rust/blockchain/system_persistence.rs [hunk 1]
```diff
@@ -74,13 +76,23 @@ impl System
...
 
-  pub fn store_to( &self, _path : &PathBuf )
+  pub fn store_to( &self, path : &PathBuf )
   {
-    /*
-    Issue : https://github.com/Learn-Together-Pro/Blockchain/issues/11
-    complexity : mid
-    stage : mid
-    */
+    let data = serde_json::to_string(self);
```
Comment (trkohler) [inline rust/blockchain/system_persistence.rs:81]: maybe this way will be easier to use in that particular case? 
```
::serde_json::to_writer(&File::create("data.json")?, &x)?
```
from https://programming-idioms.org/idiom/92/save-object-into-json-file/2347/rust

Comment (trkohler) [reply rust/blockchain/system_persistence.rs:81]: but with `panic` handle of course.

Comment (dmvict) [reply rust/blockchain/system_persistence.rs:81]: Yes, I think it is better to use example with writer. 

For example. I use writer of module `fs`:

```rs
    fs::write( path, data )
```

and handle error, of course.

**Pull request closed without merging**